bundles/org.openhab.binding.withings/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="withings"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="withingsapi">
+		<label>Withings API</label>
+		<description>This bridge represents the gateway to the Withings API.</description>
+
+		<config-description uri="thing-type:withings:bridge">
+			<parameter name="clientId" type="text">
+				<label>Client ID</label>
+				<description>Client ID provided for the application you created</description>
+				<required>true</required>
```

Comment:
The `required` tag is deprecated. You could use the attribute instead: `required="true"`

Reply:
solved